### PR TITLE
feat: add option to keep output style

### LINKED ISSUE #78: Option to keep output style
It would be nice to have an option to keep the way output is displayed (between standard, scrolled, and collapsed) since it is more part of formatting than of outputs (I am enabling scrolling when I know that the output will be large, but when I still want to display it).

### PR DESCRIPTION
**Pull request by davidxia**
with `nbstripout --keep-metadata-keys` or git config key `filter.nbstripout.keepmetadatakeys`.

closes #78

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -324,6 +324,18 @@ Additional metadata to be stripped can be configured via either
 
         --extra-keys "metadata.celltoolbar cell.metadata.heading_collapsed"
 
+You can keep certain metadata with either
+
+*   ``git config (--global/--system) filter.nbstripout.keepmetadatakeys``, e.g. ::
```
Comment (davidxia) [inline README.rst:329]: Lmk if you want something other than `keepmetadatakeys`.

**Review by davidxia — COMMENTED** [review]
Need a maintainer to approve running the tests.

File: README.rst [hunk 2]
```diff
@@ -324,6 +324,18 @@ Additional metadata to be stripped can be configured via either
...
+
+        git config --global filter.nbstripout.keepmetadatakeys '
+          cell.metadata.collapsed
+          cell.metadata.scrolled'
+
+*   the ``--keep-metadata-keys`` flag, which takes a string as an argument, e.g. ::
```
Comment (kynan) [inline README.rst:335]: space separated string

File: README.rst (hunk 1 above)
Comment (kynan) [reply README.rst:329]: No, I think this is fine :)

**Review by kynan — CHANGES_REQUESTED** [review]
Thanks, this mostly LGTM. Tests seem to be failing though, could you please fix?

**Comment by davidxia** [comment]
> Thanks, this mostly LGTM. Tests seem to be failing though, could you please fix?

Updated. Looks like I need to be approved each time to run the tests.

**Comment by kynan** [comment]
Tests still failing unfortunately :(

**Comment by davidxia** [comment]
Sorry, fixed and made sure it passes for me locally.

File: nbstripout/_nbstripout.py [hunk 3]
```diff
@@ -446,6 +449,13 @@ def main():
 
     extra_keys.extend(args.extra_keys.split())
 
+    try:
+        keep_metadata_keys = check_output((git_config if args._system or args._global else ['git', 'config']) + ['filter.nbstripout.keepmetadatakeys'], universal_newlines=True).strip().split()
```
Comment (kynan) [inline nbstripout/_nbstripout.py:453]: Can you break this line?

**Review by kynan — APPROVED** [review]

**Comment by kynan** [comment]
Ugh, test failure is now entirely unrelated to your change, caused by actions/runner-images#6399

**Comment by kynan** [comment]
"Fixed" in 4ff5477 by dropping 3.6 and adding 3.11 instead. Guess we have to go with the times...

Can you rebase this PR?

File: nbstripout/_nbstripout.py (hunk 3 above)
Comment (davidxia) [reply nbstripout/_nbstripout.py]: done

**Review by davidxia — COMMENTED** [review]
rebased

**Comment by kynan** [comment]
Thanks for your contribution!

**Comment by davidxia** [comment]
thank you!

**Pull request merged**